include/dev/MAX31855.hpp

```diff
@@ -3,6 +3,7 @@
 
 #include <EVT/io/SPI.hpp>
 #include <cstdint>
+#include <utility>
```

Comment:
What's this for?

Reply:
pretty sure I had this when I was trying to implement pair, its some useless code.